packages/ERTP/src/typeGuards.js

```diff
@@ -129,3 +129,105 @@ export const DisplayInfoShape = M.partial(
     // properties beyond those in the `base` record.
   }),
 );
+
+// //////////////////////// Interfaces /////////////////////////////////////////
+
+export const BrandShape = M.remotable('Brand');
+export const IssuerShape = M.remotable('Issuer');
+export const PaymentShape = M.remotable('Payment');
+export const PurseShape = M.remotable('Purse');
+export const DepositFacetShape = M.remotable('DepositFacet');
+const NotifierShape = M.remotable('Notifier');
+export const MintShape = M.remotable('Mint');
+
+export const BrandI = M.interface('Brand', {
+  isMyIssuer: M.callWhen(M.await(IssuerShape)).returns(M.boolean()),
+  getAllegedName: M.call().returns(M.string()),
+  getDisplayInfo: M.call().returns(DisplayInfoShape),
+  getAmountSchema: M.call().returns(M.pattern()),
+});
+
+/**
+ * @param {Pattern} [brand]
+ * @param {Pattern} [assetKind]
+ * @param {Pattern} [amountSchema]
+ */
+export const makeIssuerInterfaces = (
+  brand = BrandShape,
+  assetKind = AssetValueShape,
+  amountSchema = AmountShape,
```

Comment:
Should these parameters be renamed to shape/schema/pattern/etc. for avoiding confusion between pattern and instance of pattern?
```suggestion
  brandShape = BrandShape,
  assetValueShape = AssetValueShape,
  amountShape = AmountShape,
```

Reply:
Good point. Done.